config/helm/aws-node-termination-handler/templates/servicemonitor.yaml

```diff
@@ -25,5 +25,5 @@ spec:
   {{- end }}
   selector:
     matchLabels:
-      {{- include "aws-node-termination-handler.labels" . | nindent 6 }}
+      {{- include "aws-node-termination-handler.selectorLabelsDeployment" . | nindent 6 }}
```

Comment:
This is defined here, for reference: https://github.com/aws/aws-node-termination-handler/blob/1c61c2ddc5159350fd8d82ba812291f542a98232/config/helm/aws-node-termination-handler/templates/_helpers.tpl#L71-L74

The important point is that it defines `component` to be `deployment`.